.gitignore

```diff
@@ -47,3 +47,6 @@ docs/_build
 README.html
 
 _sandbox
+
+# JetBrains
```

Comment:
Nitpick: This belongs in your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore), not repo gitignores.